libraries/botframework-schema/src/activityEx.ts

```diff
@@ -292,6 +295,24 @@ export namespace ActivityEx {
         return isActivity(source, ActivityTypes.Handoff) ? source : null;
     }
 
+  /**
+   * Returns the source activity as an ICommandActivity object; or null, if this is not that type of activity.
+   * @param source The source activity.
+   * @returns This activity as a command activity; or null.
+   */
+     export function asCommandActivity<T>(source: Partial<Activity>): Partial<ICommandActivity<T>> {

```

Comment:
Question - does the consumer of this file have to explicitly specify a type here? I.e.,

```typescript
ActivityEx.asCommandActivity<SomeType>(...)
```

In general, if the generic type can't be inferred by the compiler I tend to do `function<T = unknown>(...): T` as it makes the function usable without passing an explicit type.

Reply:
I'm not sure. I wasn't able to find a spec for it. @EricDahlvang?

Switched all the type generics to use `<T = unknown>`, but maybe I should have only done the functions?